src/components/Icon/custom/MilestonesIcon.tsx

```diff
@@ -0,0 +1,118 @@
+            fill="white"
+          />
+          <path
+            d="M20.9371 17.877H24.3086C24.5413 17.877 24.73 17.6855 24.73 17.4493V14.0284C24.73 13.7923 24.5413 13.6008 24.3086 13.6008H20.9371C20.7043 13.6008 20.5156 13.7923 20.5156 14.0284V17.4493C20.5156 17.6855 20.7043 17.877 20.9371 17.877ZM21.3585 14.4561H23.8871V17.0217H21.3585V14.4561Z"
+            fill="white"
+          />
+          <path
+            d="M12.1719 19.7583H8.80035C8.56763 19.7583 8.37891 19.9497 8.37891 20.1859V23.6068C8.37891 23.843 8.56763 24.0344 8.80035 24.0344H12.1719C12.4046 24.0344 12.5933 23.843 12.5933 23.6068V20.1859C12.5933 19.9497 12.4046 19.7583 12.1719 19.7583ZM11.7504 23.1792H9.22178V20.6135H11.7504V23.1792Z"
+            fill="white"
+          />
+          <path
+            d="M12.4699 14.9233C12.3053 14.7563 12.0385 14.7563 11.8739 14.9233L9.98049 16.8445L9.09838 15.9494C8.93376 15.7825 8.66699 15.7825 8.50234 15.9494C8.33776 16.1164 8.33776 16.3872 8.50234 16.5542L9.68245 17.7515C9.76476 17.835 9.8726 17.8768 9.98045 17.8768C10.0883 17.8768 10.1961 17.835 10.2784 17.7515L12.4698 15.528C12.6345 15.3611 12.6345 15.0903 12.4699 14.9233Z"
+            fill="white"
+          />
+          <path
+            d="M18.5441 21.6858C18.7087 21.5188 18.7087 21.2481 18.5441 21.081C18.3795 20.9141 18.1128 20.9141 17.9481 21.081L16.0547 23.0022L15.1726 22.1071C15.008 21.9402 14.7412 21.9402 14.5766 22.1071C14.412 22.2741 14.412 22.5449 14.5766 22.7119L15.7567 23.9092C15.839 23.9928 15.9468 24.0345 16.0547 24.0345C16.1625 24.0345 16.2704 23.9928 16.3527 23.9092L18.5441 21.6858Z"
+            fill="white"
+          />
+          <path
+            d="M20.6391 22.712L21.8192 23.9093C21.9015 23.9928 22.0094 24.0346 22.1172 24.0346C22.2251 24.0346 22.3329 23.9928 22.4152 23.9093L24.6066 21.6858C24.7712 21.5188 24.7712 21.2481 24.6066 21.081C24.442 20.9141 24.1752 20.9141 24.0106 21.081L22.1172 23.0022L21.2351 22.1071C21.0704 21.9402 20.8037 21.9402 20.639 22.1071C20.4745 22.2742 20.4745 22.545 20.6391 22.712Z"
+            fill="white"
+          />
+        </g>
+        <path
+          d="M31.6694 23.5738H25.1057C24.7536 23.2761 25.3493 22.6006 25.971 22.5552C26.2413 22.535 26.5825 22.5913 26.9729 22.6734C27.0632 21.4769 27.5786 16.0994 29.5232 16.3157C31.7091 16.5579 29.5232 18.6836 28.9051 19.9429C28.4475 20.8757 27.9646 22.2128 27.7442 22.8436C27.9213 22.8818 28.1056 22.9192 28.2928 22.9538C28.5827 22.0665 29.6605 19.083 31.0311 19.2567C32.6409 19.46 31.9939 20.6565 30.8532 21.3666C30.0162 21.8884 28.989 22.6309 28.5003 22.9892C28.9138 23.0562 29.346 23.1016 29.7841 23.0915C30.9082 23.0656 31.4568 23.3964 31.6694 23.5738Z"
+          fill="#82B378"
+        />
+        {state === 'default' && (
```

Comment:
currently the `state` is used here, where `default` will add the rectangle to create the `not selected` effect while `active` and `hover` will render the original svg

<img width="1101" alt="Screen Shot 2021-04-18 at 4 27 22 PM" src="https://user-images.githubusercontent.com/32864116/115139153-f5711480-a062-11eb-92e6-cdaa6c4757ee.png">